README.md

```diff
@@ -8,7 +8,7 @@ Please see [CONTRIBUTING](https://github.com/ziadoz/awesome-php/blob/master/CONT
 ## Table of Contents
 - [Awesome PHP](#awesome-php)
     - [Composer Repositories](#composer-repositories)
-    - [Dependency Management](#dependency-management)
+    - [Dependency Management](#dependency-management)z
```

Comment:
The added character `z` should be removed.

Reply:
Yes, please remove the `z` here. Thanks.